moha/toml_tools/tools.py

```diff
@@ -53,13 +54,153 @@ def set_defaults(input_data):
             data_value = input_data[param_type][param]
             if type(data_value) == str:
                 input_data[param_type][param] = data_value.lower()
 
 
-def build_moha_moltype_1d(data):
+def build_connectivity_1d(data):
+    """
+    Builds connectivity and adjacency matrix for 1d moltype.
+
+    Parameters
+    ----------
+    data: dict
+        dict containing toml input data.
+
+    Returns
+    -------
+    connectivity: list
+        list of connected atoms of the form [("C1","C2",1),...].
+    adjacency: numpy array
+        adjacency matrix
+    """
+    norb = data["system"]["norb"]
+    connectivity = [(f"C{i}", f"C{i + 1}", 1) for i in range(1, norb)]
+    if data["system"]["bc"] == "periodic":
+        connectivity += [(f"C{norb}", f"C{1}", 1)]
+
+    adjacency = np.eye(norb, k=1)
+    if data["system"]["bc"] == "periodic":
+        adjacency[(0, -1)]
+
+    adjacency += adjacency.T
+
+    return connectivity, adjacency
+
+
+def build_connectivity_2d(data):
+    """
+    Builds connectivity and adjacency matrix for 2d moltype.
+
+    Parameters
+    ----------
+    data: dict
+        dict containing toml input data.
+
+    Returns
+    -------
+    connectivity: list
+        list of connected atoms of the form [("C1","C2",1),...].
+    adjacency: numpy array
+        adjacency matrix
+    """
+    # set Lx and Ly
+    if "Lx" not in data["system"].keys():
+        raise ValueError("2d moltype was specified but Lx was not specified")
+    Lx = data["system"]["Lx"]
+    
+    if "Ly" not in data["system"].keys():
+        raise ValueError("2d moltype was specified but Ly was not specified")
+    Ly = data["system"]["Ly"]
+
+    nsites = Lx * Ly
+    connectivity = []
+    adjacency = np.zeros((nsites, nsites))
+    for n in range(nsites):
+        nx = n %  Lx # x index
+        ny = n // Lx # y index
+        ndx = (nx + 1) % Lx # x shift
+        ndy = (ny + 1) % Ly # y shift
+        if data["system"]["bc"] != "periodic":
+            # add x neighbours to connectivity
+            if ndx != 0: # skip edge bonds on open bc
+                dn = ndx + Lx * ny
+                connectivity.append((f"C{n + 1}", f"C{dn  + 1}", 1))
+                adjacency[(n, dn)] = 1
+            # add y neighbours to connectivity
+            if ndy != 0: # skip edge bonds on open bc
+                dn = nx  + Lx * ndy
+                connectivity.append((f"C{n + 1}", f"C{dn + 1}", 1))
+                adjacency[(n, dn)] = 1
+        else:
+            # add x neighbours to connectivity
+            dn = ndx + Lx * ny
+            connectivity.append((f"C{n + 1}", f"C{dn + 1}", 1))
+            adjacency[(n, dn)] = 1
+            # add y neighbours to connectivity
+            dn = nx  + Lx * ndy
+            connectivity.append((f"C{n + 1}", f"C{dn + 1}", 1))
+            adjacency[(n, dn)] = 1
+
+    adjacency += adjacency.T
+
+    return connectivity, adjacency
```

Comment:
Before making this change, I just want to confirm that you want to remove all construction of the connectivity in all methods? This also removes atom info when building from molfile.

Reply:
Yes, I think we don't support atom information yet. It will be supported after the GSoC, but not right now. I think it's ok to remove this from everywhere, if it doesn't break things